src/store/models.d.ts

```diff
@@ -0,0 +1,22 @@
+
+export interface Letter {
+  name: string;
+  image: string;
+}
+
+export interface Word {
+  name: string;
+  image: Image;
+}
+
+export interface Image {
+  url: string;
+  isValid: boolean;
+}
+
+export interface Marker {
+  key: string;
+  position: { x: number, y: number, z: number };
+  quaternion: { x: number, y: number, z: number, w: number };
+  scale: { x: number, y: number, z: number };
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4582789)